optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -984,10 +1013,18 @@ def __init__(
         *,
         optuna_seed: Optional[int] = None,
     ) -> None:
+        if callbacks is None:
+            callbacks = []
+
+        if verbose_eval is True:
+            callbacks.append(lgb.log_evaluation(show_stdv=show_stdv))
+        elif isinstance(verbose_eval, int):
+            callbacks.append(lgb.log_evaluation(period=verbose_eval, show_stdv=show_stdv))
+
         super().__init__(
             params,
             train_set,
-            num_boost_round,
+            num_boost_round=num_boost_round,
```

Comment:
I think it would be better to align the order of arguments with the order in the `__init__` function.
```suggestion
            callbacks=callbacks,
            num_boost_round=num_boost_round,
```

Reply:
Thanks for the comment. I fixed them.